invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/icon.overrides

```diff
@@ -19,4 +19,10 @@
     border-color: @errorTextColor ;
     color: @errorTextColor;
   }
+
+  &.carousel-arrow {
```

Comment:
could you attach a screenshot?

Reply:
thanks! 
in this case, because this is zenodo theme, is there any way to define the styling of this component (the whole carousel) for app-rdm, and move the zenodo specific styling to zenodo-rdm? WDYT?

Reply:
disscused, agreed to leave it as it is